src/pages/cookie-settings.mdx

```diff
@@ -0,0 +1,76 @@
+
+:::note
+
+<ManageCookieSettings />
+
+:::
+
+### What are cookies?
+
+Cookies are small text files that are used to store small pieces of information. The cookies are stored on your device when the website is loaded on your browser. These cookies help us make the website function properly, make the website more secure, provide better user experience, and understand how the website performs and to analyze what works and where it needs improvement.
+
+### How do we use cookies?
+
+As most of the online services, our website uses cookies first-party and third-party cookies for a number of purposes. The first-party cookies are mostly necessary for the website to function the right way, and they do not collect any of your personally identifiable data.
+
+The third-party cookies used on our websites are used mainly for understanding how the website performs, how you interact with our website, keeping our services secure, providing advertisements that are relevant to you, and all in all providing you with a better and improved user experience and help speed up your future interactions with our website.
+
+The below list details the cookies used in our website.
+<table>
+  <thead>
+    <tr>
+      <th scope="col">Cookie</th>
+      <th scope="col">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th colspan="2">Necessary</th>
+    </tr>
+    <tr>
+      <td>cookie-preferences</td>
+      <td>Ironically, to respect your cookie preferences we have to set a cookie for storing and processing them.</td>
+    </tr>
+    <tr>
+      <th colspan="2">Analytics</th>
+    </tr>
+    <tr>
+      <td>_ga</td>
+      <td>The _ga cookie, installed by Google Analytics, calculates visitor, session and campaign data and also keeps track of site usage for the site's analytics report. The cookie stores information anonymously and assigns a randomly generated number to recognize unique visitors.</td>
+    </tr>
+    <tr>
+      <td>_gid</td>
+      <td>Installed by Google Analytics, _gid cookie stores information on how visitors use a website, while also creating an analytics report of the website's performance. Some of the data that are collected include the number of visitors, their source, and the pages they visit anonymously.</td>
+    </tr>
+    <tr>
+      <td>_gat_gtag_UA_159566509_1</td>
+      <td>Set by Google to distinguish users.</td>
+    </tr>
+    <tr>
+      <td>_sp5_</td>
+      <td>The _sp5_ cookie, installed by Snowplow Analytics, stores information on how visitors use a website, while also creating an analytics report of the website's performance. Some of the data that are collected include the number of visitors, their source, and the pages they visit anonymously.</td>
+    </tr>
+  </tbody>
+</table>
+
+### How can I control the cookie preferences ?
+
+You can manage your cookies preferences by clicking on the "Settings" button and enabling or disabling the cookie categories on the popup according to your preferences.
```

Comment:
either change this to “manage your consent”, or change the widget to say “settings” (which I think sounds more friendly)